### DISCUSSION TITLE
Port 8080

### DISCUSSION THREAD

**Discussion by ekjuanrejon** [Aguafrommars/TheIdServer#1169] [General]
[15:06:51 Information] Microsoft.Hosting.Lifetime
2024-01-14 15:06:51 Now listening on: http://[::]:8080

How to I change it to port 80... running inside tye

**Comment by aguacongas** [comment]
How do you launch it ?

**Comment by ekjuanrejon** [comment]
docker image from docker hub
using the the following tag: 8.0.0-preview1-0001

**Reply by aguacongas** [reply]
Why do you want to change the internal port ?
